src/AudioFileSourceHTTPStream.cpp

```diff
@@ -44,12 +86,37 @@ bool AudioFileSourceHTTPStream::open(const char *url)
 #ifndef ESP32
   http.setFollowRedirects(HTTPC_FORCE_FOLLOW_REDIRECTS);
 #endif
+  const char* headers[] = { "Transfer-Encoding" };
+  http.collectHeaders( headers, 1 );
   int code = http.GET();
   if (code != HTTP_CODE_OK) {
     http.end();
     cb.st(STATUS_HTTPFAIL, PSTR("Can't open HTTP request"));
     return false;
   }
+  if (http.hasHeader("Transfer-Encoding")) {
+    audioLogger->printf_P(PSTR("Transfer-Encoding: %s\n"), http.header("Transfer-Encoding").c_str());
+    if(http.header("Transfer-Encoding") == String("chunked")) {
```

Comment:
Minor, but can we make the constant `String()`s use `PSTR()` for the values?  On the 8266 that saves RAM, which is always super tight...

Reply:
the http.header returns a String object, how do you suggest to compare them? with c_str or something?

Reply:
`String(PSTR("chunked"))` is what I meant here

Reply:
Fixed !